src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -836,10 +840,9 @@ public boolean isFeedbackParticipantVisible(boolean isGiver, FeedbackResponseAtt
             isVisible = visibilityTable.get(responseId)[Const.VISIBILITY_TABLE_RECIPIENT];
             participantType = question.recipientType;
         }
-        boolean isTypeSelf = participantType == FeedbackParticipantType.SELF;
         boolean isTypeNone = participantType == FeedbackParticipantType.NONE;
 
-        return isVisible || isTypeSelf || isTypeNone;
+        return isVisible || isTypeNone;
```

Comment:
Here

Reply:
This is the bug described in #8203. Before the change, self feedback question would always be visible regardless of the settings.